### PR TITLE
Use upstream 1.6.9 for blacklist plugin tesing

### PR DESCRIPTION
**Pull request by brinker211**
PR to add values change to testing branch `addCustomPlugin` during Q1 hackathon

### CODE REVIEW

File: helm/coredns-app/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 1.6.5
+appVersion: 1.6.9
```
Comment (sslavic) [inline helm/coredns-app/Chart.yaml]: actual tag reference https://github.com/giantswarm/coredns-app/blob/master/helm/coredns-app/values.yaml#L31 has to be changed too

**Comment by brinker211** [comment]
The build fails on:
 `Job 'e2eTestBasic' requires 'push-to-default-app-catalog', which is the name of 0 other jobs in workflow 'build_e2e'` 

from Circleci. Is there any specific integration testing that needs to be left in the `.circleci/config.yml`? Otherwise I can just leave the `push-coredns-to-incubator` rule but comment out all the e2e tests to allow circleci to proceed and test. TIA for clarification.

File: helm/coredns-app/Chart.yaml (hunk 1 above)
Comment (brinker211) [reply helm/coredns-app/Chart.yaml]: Thanks @sslavic . What I am trying to accomplish is to test a custom coredns image (hackathon project) with our GS helm chart values in a TC. The changes I have made:

- Create new test branch "AddCustomPlugin" and using this PR to track changes with @stone-z .
    - change `chart.yaml` to the same upstream version used in the "new" image
    - change  `values.yml` values `giantswarm/coredns:1.6.5` to `benrain/blacklist:v1` newly compiled coredns image on dockerhub
    - update `.cicleci/config.yml` to put this .tar for custom helm chart into `default-test-catalog` in order for TC to be able to pull the new test image

Comment (sslavic) [reply helm/coredns-app/Chart.yaml]: ah, clear, thanks for heads up!
maybe consider also contributing actual coredns 1.6.9 update too :)

**Comment by stone-z** [comment]
I don't think we need this anymore. Plugin has its own CI now https://github.com/giantswarm/coredns-warnlist-plugin

**Pull request closed without merging**